packages/template-phpstan-compiler/src/Reporting/TemplateErrorsFactory.php

```diff
@@ -68,4 +76,15 @@ private function resolveNearestPhpLine(array $phpToTemplateLines, int $desiredLi
 
         return $lastTemplateLine;
     }
+
+    private function resolveTipForMessage(string $message): ErrorMessageWithTip
+    {
+        $tip = '';
+        $undefinedFilter = str_replace('Access to an undefined property Latte\Runtime\FilterExecutor::$', '', $message);
+        if ($undefinedFilter !== $message) {
+            $message = 'Undefined filter ' . $undefinedFilter;
+            $tip = 'Register it in parameters > latteFilters. See https://github.com/symplify/symplify/tree/main/packages/phpstan-latte-rules#configuration';
```

Comment:
Looks good. But in what file should be registered? :)

Reply:
I don't know for now :) I need 2-3 more examples of such messages which need some transformation and tip to see how to implement it universally. Do you have some ideas?

Reply:
I would say `phpstan.neon` is the one

Reply:
I'm not convinced with phpstan.neon, what about this way https://github.com/symplify/symplify/pull/3755/commits/424e909e3f20b29bee16e97e0045030865112fc3?

Reply:
Still missed the file. That's the first thing people look for while seeing "add somethiing to file".
E.g. "register service" could go many files. Mentioning the exact one will remove thought like we have in this dicussion :)

Reply:
Oh, you mean update the tip message :) I thought you want to setup these new messages and tips in phpstan.neon...

Reply:
Yes, that's what I mean :+1: 

Btw, looking at original description, what DX problem is this improvement trying to solve? It might take much manual work to duplicate all non-basic filters/macros just to satisfy PHPStan :thinking: 

I've seen other PHPStan extensions like Doctrine and Symfony to link the entity manager file factory or Symfony service container, just to provide internal project data.

Maybe it would be useful to add Latte engine factory or something, so the PHPStan has access to the same Engine like project has.

Reply:
I've added phpstan.neon to the description

> Btw, looking at original description, what DX problem is this improvement trying to solve? 

It is trying to solve problem when user of this extension gets error which is totally strange. 

> It might take much manual work to duplicate all non-basic filters/macros just to satisfy PHPStan 🤔
> I've seen other PHPStan extensions like Doctrine and Symfony to link the entity manager file factory or Symfony service container, just to provide internal project data.
> 
> Maybe it would be useful to add Latte engine factory or something, so the PHPStan has access to the same Engine like project has.

This is not part of this PR, but yes - you have to somehow register all filters.
I've prepared automatic way for registering all used latte filters from project. I can do it for our projects using DI container because I know we have app/bootstrap.php file which returns Container and this [package](https://github.com/ricco24/helpers) . But how to implement it for other users? Is it possible without Container?

My code for this:
```
<?php

$bootstrapPath = getcwd() . '/app/bootstrap.php';
if (!file_exists($bootstrapPath)) {
    return [];
}

$container = require $bootstrapPath;
$filterLoader = $container->getByType('Kelemen\Helper\Nette\FilterLoader', false);

if (!$filterLoader) {
    return [];
}

if (!method_exists($filterLoader, 'getFilters')) {
    return [];
}

$filters = [];
foreach ($filterLoader->getFilters() as $name => $filter) {
    if (is_string($filter)) {
        $filters[$name] = $filter;
        continue;
    }
    if (is_array($filter) && count($filter) === 2) {
        $class = $filter[0];
        if (is_object($class)) {
            $class = get_class($class);
        }
        $method = $filter[1];
        $filters[$name] = [$class, $method];
    }
}

return [
    'parameters' => [
        'latteFilters' => $filters,
    ],
];
```